deploy/deploy.go

```diff
@@ -51,6 +52,7 @@ type Deployer struct {
 
 	target        *target          // the target to deploy to
 	matchers      []*matcher       // matchers to apply to uploaded files
+	mediaTypes    *media.Types     // Hugo's MediaType to guess ContentType
```

Comment:
Why is this a pointer? A pointer to a slice is not something I use very often.

Reply:
Because I misunderstood this would improve performance.
But I have found it false.
Fixed, thanks